static/app/views/replays/detail/trace/index.tsx

```diff
@@ -18,10 +20,43 @@ function PerfDisabled() {
   );
 }
 
+type EventHandler = (...args: any[]) => void;
+
+class EventEmitter {
+  private events: {[key: string]: EventHandler[]} = {};
+
+  on(event: string, handler: EventHandler): void {
```

Comment:
```suggestion
  on(event:'replay timestamp change', handler: EventHandler): void {
```

Reply:
I would also prefer if we had stronger types for this. You can copy the dispatching reducer event emitter declarations or the profiling flamegraph event emitter to see how to do that